README.md

```diff
@@ -1,3 +1,28 @@
+# 1. Introduction
+
+This web application is a [Google Summer of Code 2019](https://summerofcode.withgoogle.com) project under the [MacPorts](https://www.macports.org) organisation.
+
+**Student**: [Arjun Salyan](https://github.com/arjunsalyan) <br>
+**Mentors**: [Mojca Miklavec](https://github.com/mojca), [Umesh Singla](https://github.com/umeshksingla) <br>
+
+### Architecture of the Project
+
+The web application is based on the [Django Framework](http://djangoproject.com) utilising a PostgreSQL database.
+
+The app is supposed to be deployed in a docker container with nginx and uWSGI serving the content.
+
+**Demo**: [AWS EC2](http://ec2-52-34-234-111.us-west-2.compute.amazonaws.com) (running a docker container)
+
+### Goal of the Project
+
+Build an all-in-one application that:
```

Comment:
I would make this the first paragraph. I would not start as "Build an ...", but rather something like "This is a web application which:". The points below should probably all start with a capital letter (I have a feeling these are some strange rules of the English grammar, but I'm not absolutely sure).